internal/services/netapp/netapp_volume_resource.go

```diff
@@ -452,7 +447,7 @@ func resourceNetAppVolumeCreateUpdate(d *pluginsdk.ResourceData, meta interface{
 
 		// Wait for volume replication authorization to complete
 		log.Printf("[DEBUG] Waiting for replication authorization on NetApp Volume Provisioning Service %q (Resource Group %q) to complete", id.Name, id.ResourceGroup)
```

Comment:
`log.Printf("[DEBUG] Waiting for replication authorization on NetApp Volume Provisioning Service %s", id)`